src/Cursors.mockClient.test.ts

```diff
@@ -20,6 +20,47 @@ interface CursorsTestContext {
   fakeMessageStub: Types.Message;
 }
 
+const self: SpaceMember = {
+  connectionId: 'connectionId1',
+  clientId: 'clientId1',
+  isConnected: true,
+  profileData: {},
+  location: {},
+  lastEvent: { name: 'enter', timestamp: 0 },
+};
+
+const allCursors = {
```

Comment:
let's put `self` and `allCursors` into the `beforeEach<CursorsTestContext>((context)` block, so they are not shared between all tests.